test/TestServerProjectsLowLevel/dpg_update1/Custom/ParamsClient.cs

```diff
@@ -0,0 +1,320 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable disable
+
+using System;
+using System.Threading.Tasks;
+using Azure;
+using Azure.Core;
+
+namespace dpg_update1_LowLevel
+{
+    /// <summary> The Params service client. </summary>
+    [CodeGenSuppress("PostParametersAsync", typeof(RequestContent), typeof(RequestContext))]
+    [CodeGenSuppress("PostParameters", typeof(RequestContent), typeof(RequestContext))]
+    [CodeGenSuppress("CreatePostParametersRequest", typeof(RequestContent), typeof(RequestContext))]
+    public partial class ParamsClient
+    {
+        /// <summary>
+        /// Get true Boolean value on path.
+        ///  Initially has one optional query parameter. After evolution, a new optional query parameter is added
+        ///  Overload method to be compatible with dpg_initial where optional parameters become required.
+        /// </summary>
+        /// <param name="optionalParam"> I am an optional parameter. </param>
+        /// <param name="context"> The request context, which can override default behaviors on the request on a per-call basis. </param>
+        public virtual async Task<Response> GetOptionalAsync(string optionalParam, RequestContext context)
```

Comment:
Are we able to ExportAPI.cs for these clients?  It would be helpful for reviewing the public APIs for them.